dl4j-cv-labs/src/main/java/global/skymind/training/object_detection/AvocadoBananaDetector/AvocadoBananaDetector_TinyYOLO.java

```diff
@@ -111,8 +74,8 @@ public static void main(String[] args) throws Exception {
             //     STEP 2.3: Transfer Learning steps - Modify prebuilt model's architecture
 //            model = getNewComputationGraph(pretrained, priors, fineTuneConf);
 //            System.out.println(model.summary(InputType.convolutional(
-//                    FruitDataSetIterator.yoloheight,
-//                    FruitDataSetIterator.yolowidth,
+//                    MetalDefectDataSetIterator.yoloheight,
```

Comment:
please recheck your codebase, looks like the replace overwriting the other files.

Reply:
done